data_handler/handlers/order_books/myswap/main.py

```diff
@@ -121,87 +112,76 @@ async def _calculate_order_book(self, pool_id: str) -> None:
         asks, bids = data[data["tick"] >= current_tick], data[data["tick"] < current_tick]
         bids = bids.sort_values("tick", ascending=True)
 
         # Add asks and bids to the order book
-        self.add_bids(bids, (min_tick, max_tick))
-        self.add_asks(asks, bids.iloc[0]["liq"], (min_tick, max_tick))
-        tvl = (
-                (sum([bid[1] for bid in self.bids]) * self._usd_price) +
-                (sum([ask[1] for ask in self.asks]) * self._usd_price)
-        )
+        self.add_bids(pool_bids=bids)
+        self.add_asks(pool_asks=asks, pool_liquidity=bids.iloc[0]["liq"])
 
-    def add_asks(self, pool_asks: pd.DataFrame, pool_liquidity: Decimal, price_range: tuple[Decimal, Decimal]) -> None:
+    def add_asks(self, pool_asks: pd.DataFrame, pool_liquidity: Decimal) -> None:
         """
         Add asks data to the order book.
         :param pool_asks: pd.DataFrame - Asks in the pool.
         :param pool_liquidity: Decimal - The pool liquidity.
-        :param price_range: tuple[Decimal, Decimal] - The price range for filtering.
         """
         if pool_asks.empty:
             return
         local_asks = []
-        prev_tick = Decimal(pool_asks.iloc[0]['tick'].item())
-        prev_price = self.tick_to_price(prev_tick)
+        next_tick = Decimal(pool_asks.iloc[0]['tick'].item())
+        next_price = self.tick_to_price(next_tick)
         y = self._get_token_amount(
-            pool_liquidity,
-            self.current_price.sqrt(),
-            prev_price.sqrt(),
+            current_liq=pool_liquidity,
+            current_sqrt=self.current_price.sqrt(),
+            next_sqrt=next_price.sqrt(),
             is_ask=False
         )
-        local_asks.append((prev_price, y))
+        local_asks.append((next_price, y))
         for index, bid_info in enumerate(pool_asks.iloc[::-1].itertuples(index=False)):
             if index == 0:
                 continue
             current_tick = Decimal(pool_asks.iloc[index - 1]['tick'].item())
             current_price = self.tick_to_price(current_tick)
             y = self._get_token_amount(
-                Decimal(pool_asks.iloc[index - 1]['liq'].item()),
-                current_price.sqrt(),
-                Decimal(self.tick_to_price(pool_asks.iloc[index]['tick'].item())).sqrt(),
+                current_liq=Decimal(pool_asks.iloc[index - 1]['liq'].item()),
+                current_sqrt=current_price.sqrt(),
+                next_sqrt=Decimal(self.tick_to_price(pool_asks.iloc[index]['tick'].item())).sqrt(),
                 is_ask=False
             )
             local_asks.append((current_price, y))
-        if self.apply_filtering:
-            self.asks.extend([ask for ask in local_asks if price_range[0] < ask[0] < price_range[1]])
-            return
         self.asks.extend(local_asks)
 
-    def add_bids(self, pool_bids: pd.DataFrame, price_range: tuple[Decimal, Decimal]) -> None:
+    def add_bids(self, pool_bids: pd.DataFrame) -> None:
```

Comment:
How come this method doesn't need the `pool_liquidity` parameter?

Reply:
As we start to process bids there are no cumulative liquidity in the pool, so it shouldn't be needed. Understood while refactoring